lib/linter/linter.js

```diff
@@ -859,6 +860,7 @@ function runRules(sourceCode, configuredRules, ruleMapper, parserOptions, parser
                 getDeclaredVariables: sourceCode.scopeManager.getDeclaredVariables.bind(sourceCode.scopeManager),
                 getCwd: () => cwd,
                 getFilename: () => filename,
+                getPhysicalFilename: () => physicalFilename || filename,
```

Comment:
I don’t think it’s possible for physicalFilename to be missing based on where it’s passed in. Can you double check?

Reply:
Yeah, needed to update one more instance in `normalizeVerifyOptions`.  Should be good now.